mappings/net/minecraft/world/gen/feature/PlacedFeature.mapping

```diff
@@ -18,6 +18,18 @@ CLASS net/minecraft/class_6796 net/minecraft/world/gen/feature/PlacedFeature
 		ARG 2 random
 		ARG 3 pos
 	METHOD method_39650 generate (Lnet/minecraft/class_5281;Lnet/minecraft/class_2794;Ljava/util/Random;Lnet/minecraft/class_2338;)Z
+		COMMENT Generates a configured feature at the positions obtained by applying
+		COMMENT placement modifiers to the given position.
+		COMMENT
+		COMMENT <p>The steps to calculate positions are as follows:
+		COMMENT <ul>
+		COMMENT   <li>applying the first modifier to the given position,
+		COMMENT   <li>applying the second modifier to each of the result positions,
+		COMMENT   <li>flattening the result positions,
+		COMMENT   <li>applying the third modifier to each of the result positions,
+		COMMENT   <li>flattening the result positions,
+		COMMENT   <li>and so on.
```

Comment:
Do you really need to explain how `flatMap` works here?

Reply:
Probably mentioning `flatMap` just works. I just wanted to avoid confusion with the 1.17 decorator. Decorator was a (linked-list backed) stack, which consumes the last set decorator first.

Reply:
Don't think a list is appropriate here as you are generally repeating one procedure. You should say something like:
"starting with a set of one position, for each placement modifier, we send each position in the set into the modifier and take union of the results to obtain the updated set of positions"